core/network/impl/synchronizer_impl.cpp

```diff
@@ -354,24 +387,30 @@ namespace kagome::network {
         or block_tree_->has(header.parent_hash);
 
     if (parent_is_known) {
-      loadBlocks(peer_id, block_info, [wp{weak_from_this()}](auto res) {
-        if (auto self = wp.lock()) {
-          SL_TRACE(self->log_, "Block(s) enqueued to apply by announce");
-        }
-      });
+      for (const auto &p_id : selected_peers) {
+        loadBlocks(p_id, block_info, [wp{weak_from_this()}](auto res) {
+          if (auto self = wp.lock()) {
+            SL_TRACE(self->log_, "Block(s) enqueued to apply by announce");
```

Comment:
res contains either information about block's number and hash that was downloaded or error.
This information could be added to log message